CRM/Civicase/Hook/BuildForm/AddCaseCategoryInstanceField.php

```diff
@@ -19,6 +19,7 @@ public function run(CRM_Core_Form &$form, $formName) {
     }
 
     $this->addCategoryInstanceFormField($form);
+    $this->addSingularLabelFormField($form);
```

Comment:
I think we should not add this function here. The class name states that this class adds a case category instance field. We should either add another class to manage this field or modify the name of the class to reflect what it does  but tbh it would be better to add another class to manage this field.